### DISCUSSION TITLE
Add "Edit Collection" on left panel navigation

### DISCUSSION THREAD

**Discussion by Kematia** [directus/directus#4732] [Feature Requests]
Bom dia, I was thinking that something like this:

![image](https://user-images.githubusercontent.com/17851386/112841716-1c11e000-90a1-11eb-8779-f256924ddec4.png)

could be good DX. Let me know if you guys like it, I can open a PR.

**Comment by benhaynes** [comment]
I don't mind this — _Right-Click_ seems like a good balance. @rijkvanzanten — thoughts on this approach?

**Reply by rijkvanzanten** [reply]
Makes sense to me!

Just make sure it's only shown to admin users though 🙂

**Comment by jaads** [comment]
Heya! Thank you for taking the time to submit this request! 

It has been over **90 days**, and this discussion has not received at least **15 votes** from the community. This means that we don't feel like there's enough community interest to warrant further R&D into this topic at this time. :ice_cube:

This request will now be closed to keep our discussions tidy. Please reach out if you have any questions!

_For more information, see our [Feature Request Process](https://docs.directus.io/contributing/feature-request-process.html)._
